_docs_integrate/request-and-response-introduction.md

```diff
@@ -138,7 +138,9 @@ Depending on whether the ProposeAttributeRequestItem is to be accepted or reject
 
 After the Recipient has responded to the ProposeAttributeRequestItem, a suitable [ResponseItem]({% link _docs_integrate/data-model-overview.md %}#responseitems) is generated and transferred to the Sender of the Request:
 
-- After accepting this RequestItem, a [ProposeAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#proposeattributeacceptresponseitem) will be transferred.
+- After accepting this RequestItem with an Attribute that wasn't shared with the peer before, a [ProposeAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#proposeattributeacceptresponseitem) will be transferred.
```

Comment:
- Emphazise that this is a new Attribute or an existing Attribute (that neither itself nor one of its potentially existing predecessors was shared with the peer before).
- Also emphazise that none of the potentially existing predecessor of the existing Attribute was shared with the peer before when a ProposeAttributeAcceptResponseItem is generated.